### ISSUE TITLE
Details for reproducing

### ISSUE THREAD

**Issue by KSXGroup** [Sigta678/Social-SSL#3]
Hi, I am trying to reproduce your result on eth dataset but it is far away from the reported result (just better than linear).

I preprocessed from raw data using your data converter and pretrained for 700 epoch (lr=3e-6) + 300 epoch (lr=3e-7) with lambda value you mentioned in paper. And then finetuned on eth. But I could not got the reported value. Maybe I missed some steps. So could you provide detail for reproducing or trained model checkpoint ? 

Thanks!

**Comment by Sigta678** [comment]
Hi @KSXGroup,

Thanks for your question.
The missing steps might be the learning rate decay on the fine-tuning process, or you can try to drop some data and leave just 1% or 10% for fine-tuning. We discussed this part in our experiment "amount of data for fine-tuning". Also, you can check our Figure 2 in supplementary materials, this explains the phenomenon you met on eth.
<img width="695" alt="Appendix_Fig2" src="https://user-images.githubusercontent.com/35592565/210472937-f27165f6-9ea3-44cb-bdc4-83490e91e303.png">

Hope this helps!
